gwnr/waveform/esigma_utils.py

```diff
@@ -616,6 +616,10 @@ def get_imr_esigma_modes(
                 )
                 * 2
             )  # Extra 2-factor for returning the (2,2)-mode frequency
+            if verbose:
+                print(
+                    f"""Location 2: f_window_mr_transition = {f_window_mr_transition}"""
+                )
```

Comment:
"Location 2" seems a bit random given that there are no "Location" numbers specified in other error messages...

Reply:
agreed, this was an oversight. removed.